### PR TITLE
Clean up persistence (a step towards persistence for transaction blocker)

### PR DESCRIPTION
**Pull request by kjarosh**

### CODE REVIEW

File: modules/common/src/main/kotlin/com/github/davenury/common/persistence/JedisPersistence.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        redis.call('SET', KEYS[1], ARGV[2])
+    end
+
+    return witness
+"""
+
+/**
+ * @author Kamil Jarosz
+ */
+class JedisPersistence(host: String, port: Int): Persistence {
```
Comment (Davenury) [inline modules/common/src/main/kotlin/com/github/davenury/common/persistence/JedisPersistence.kt:18]: Jedis is a client - I'd rename it to "Redis"

File: src/main/kotlin/com/github/davenury/ucac/common/PersistenceFactory.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import com.github.davenury.common.persistence.JedisPersistence
+import com.github.davenury.common.persistence.Persistence
+import com.github.davenury.ucac.Config
+import com.github.davenury.ucac.PersistenceType
+
+/**
+ * @author Kamil Jarosz
+ */
+class PersistenceFactory {
+    fun createForConfig(config: Config): Persistence {
```
Comment (Davenury) [inline src/main/kotlin/com/github/davenury/ucac/common/PersistenceFactory.kt:14]: Such method could be also done in interface - why create such class?

Comment (kjarosh) [reply src/main/kotlin/com/github/davenury/ucac/common/PersistenceFactory.kt:14]: Due to dependency issues, `PersistenceFactory` is in a different module than `Persistence`.

File: modules/common/src/main/kotlin/com/github/davenury/common/persistence/JedisPersistence.kt (hunk 1 above)
Comment (kjarosh) [reply modules/common/src/main/kotlin/com/github/davenury/common/persistence/JedisPersistence.kt:18]: Renamed

**Review by Davenury — APPROVED** [review]

**Pull request merged**